### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by LavaToaster**
I did a sneaky and updated the config file to use the class syntax, but if you don't want that then that's fine, I'll be happy to remove it.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please revert the changes to the config file.

**Comment by GrahamCampbell** [comment]
Could you also import things in the tests, rather than a global escape. Thanks. If the tests pass after that, I'll merge. :)

**Comment by LavaToaster** [comment]
Suggestion: Can you make style ci fail on that?

**Comment by LavaToaster** [comment]
Not sure how I missed the imports, considering they're literally used two lines above, but thanks for catching that. I do import them in my own code. I was probably thrown of by the use of the global escape in the docblock.

Anyway, I have trashed the config change and squashed everything together. :)

File: tests/Displayers/DebugDisplayerTest.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function testServerError()
 
         $response = $displayer->display(new Exception('Down for maintenance!'), 'foo', 503, []);
 
-        $this->assertInternalType('string', $response->getContent());
+        $this->assertIsString($response->getContent());
```
Comment (GrahamCampbell) [inline tests/Displayers/DebugDisplayerTest.php]: This method does not exist in PHPUnit 6.5, which is why the tests are failing.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
Thanks @Lavoaster. :)

**Pull request merged**